apple/Sources/FerrostarMapLibreUI/Extensions/View.swift

```diff
@@ -0,0 +1,13 @@
+import SwiftUICore
+
+extension View {
+    /// Do something reasonable-ish for clients that don't yet support
+    /// safeAreaPadding - in this case, fall back to regular padding.
+    func safeAreaPaddingPolyfill(_ insets: EdgeInsets) -> AnyView {
+        if #available(iOS 17.0, *) {
+            AnyView(self.safeAreaPadding(insets))
+        } else {
+            AnyView(padding(insets))
+        }
```

Comment:
BTW, as an alternative, we could just use `padding` on all platforms, rather than `safeAreaPadding`, but `safeAreaPadding` is cool when available because you can do things like this:

<img width="708" alt="Screenshot 2024-09-18 at 20 58 37" src="https://github.com/user-attachments/assets/4ff2745b-f01b-4eec-9eeb-b5d39936ec85">

See how the InstructionView spills out past the safeArea? It's not possible with `padding` (egregious hacks like hardcoded negative padding maybe not withstanding).

Currently we don't have a use case for this, but since the grid is intended to be configurable, I think this gives users more flexibility if they want some kind of edge-to-edge thing in their overlays.